File: DailyFlash Solutions/DailyFlash Solutions/58-DailyFlash_Solutions/27_March_Solution_Three/Python/prog5.py
import math
x1 = int(input("x1 : "))
y1 = int(input("y1 : "))
x2 = int(input("x2 : "))
y2 = int(input("y2 : "))
x3 = int(input("x3 : "))
y3 = int(input("y3 : "))
bs = int(input("base : "))
a = math.sqrt((y1 - y2) * (y1 - y2) + (x1 - x2) * (x1 - x2))
b = math.sqrt((y3 - y2) * (y3 - y2) + (x3 - x2) * (x3 - x2))
c = math.sqrt((y1 - y3) * (y1 - y3) + (x1 - x3) * (x1 - x3))
s = (a + b + c)/ 2
print("Ht : ", math.sqrt(s * (s - a) * (s - b) * (s - c)) * 2 / bs)

File: DailyFlash Solutions/DailyFlash Solutions/58-DailyFlash_Solutions/27_March_Solution_Three/C/prog5.c
#include<stdio.h>
#include<math.h>
int main(void) {
	float x1, x2, y1, y2, x3, y3, a, b, c;
	printf("(A)Enter x1 : ");
	scanf("%f", &x1);
	printf("(A)Enter y1 : ");
	scanf("%f", &y1);
	printf("(B)Enter x2 : ");
	scanf("%f", &x2);
	printf("(B)Enter y2 : ");
	scanf("%f", &y2);
	printf("(C)Enter x3 : ");
	scanf("%f", &x3);
	printf("(C)Enter y3 : ");
	scanf("%f", &y3);
	float bs;
	printf("Base : ");
	scanf("%f", &bs);
	a = sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1));
	b = sqrt((x2 - x3) * (x2 - x3) + (y2 - y3) * (y2 - y3));
	c = sqrt((x3 - x1) * (x3 - x1) + (y3 - y1) * (y3 - y1));
	float s = (a + b + c)/ 2;
	printf("Ht : %f", sqrt(s * (s - a) * (s - b) * (s - c)) * 2 / bs);

	
}
